### DISCUSSION TITLE
I wish the knowledge graph had...

### DISCUSSION THREAD

**Discussion by Waidhoferj** [calpoly-csai/cal-poly-knowledge-graph#17] [Ideas]
What information should we add to the Cal Poly Knowledge Graph? If you find yourself constantly looking for a campus resource, its probably a good thing to add to the API. Give a little description of what you'd add and how you'd find it useful.

**Comment by Waidhoferj** [comment]
I wish the knowledge graph had information from the Campus COVID-19 dashboard so that I can make apps that track campus COVID numbers.

**Comment by PAenugula** [comment]
I wish the knowledge graph could recommend me courses based on my interests/past courses.

**Comment by ryanchansf** [comment]
I wish the knowledge graph could give information about professor TA openings and research/project opportunities.

**Comment by sarahEllwein** [comment]
I wish the knowledge graph had UPD activity logs so I can create a dashboard on campus safety.

**Reply by Waidhoferj** [reply]
[Here](https://afd.calpoly.edu/police/campus-reports/logs) is the site to scrape to get this info in the knowledge graph
